core/src/main/java/bisq/core/trade/TradeManager.java

```diff
@@ -543,6 +544,14 @@ public void removePreparedTrade(Trade trade) {
 
     private void removeTrade(Trade trade) {
         tradableList.remove(trade);
+
+        // collect all node addresses (hidden services) we still need for active offers/trades
```

Comment:
isnt that exactly the same code as above?

Reply:
in OpenOfferManager you mean? yes, somewhat.